constitution.tex

```diff
@@ -190,9 +190,8 @@
 \asubsection{Active Membership Selection}
 Qualifying members may self-select into Active Membership by paying dues to the Financial Director and notifying the Evaluations Director.
 \\*\\*
-Any Alumni Member in bad standing (\ref{Alumni Membership Selection}) may become an Active Member by notifying the Evaluations Director of their intent to participate, who will bring them up for vote at the next Evaluations meeting.
-If a majority is in favor of the return, the Alumni is reinstated as an Active Member with Off-Floor status effective immediately.
-The decision will be announced at the following House meeting.
+Any Alumni Member in bad standing (\ref{Alumni Membership Selection}) may become an Active Member by notifying the Evaluations Director of their intent to participate, who will bring them up for vote at the next House meeting.
+If Fifty Percent of Active Members present are in favor of the return (as defined in \ref{Fifty Percent}), the Alumni is reinstated as an Active Member with Off-Floor status effective immediately.
```

Comment:
Would you rewrite this to use the voting constructs we normally use? I believe "Simple Majority" is the more commonly referenced construct.

Reply:
Yea I talked to marry and Galen about this it is pretty inconsistent how voting is referenced in the constitution, I simply forgot simple majority is a way to do it. I eventually want to go back and standardize all other reference's. I like the use of simple majority and will change that .